experiments/compositions/composition/tests/scenario/scenario.go

```diff
@@ -69,6 +70,8 @@ func New(t *testing.T, dataFolder string) *Scenario {
 	if dataFolder == "" {
 		dataFolder = "../../tests/data/" + t.Name()
 	}
+	// TODO (barney-s) parameterize or make it global
+	logRoot := "../../"
```

Comment:
I suggest putting the logger into the context if you can.  (Not a blocker)

Reply:
Edit: although I now see this is more about where to dump container logs ... let's see how it goes!

Reply:
The issue with Tests is i cannot setup a context in Testmain and use it in Testcases. This is a config that belong in TestMain. Didnt want to set it in every testcase. Hence the comment about making it a global var.